docs/reference/system-requirements.rst

```diff
@@ -56,12 +56,11 @@ GPU is not listed on this table, it's not officially supported by AMD.
       "AMD Radeon RX 7900 GRE", "RDNA3", "gfx1100", "✅"
       "AMD Radeon VII", "GCN5.1", "gfx906", "⚠️"
 
-✅: **Supported** - AMD enables these GPUs in our software distributions for the corresponding
-ROCm product.
+✅: **Supported** - Official software distributions of the current ROCm release fully enables this hardware.
```

Comment:
```suggestion
✅: **Supported** - Official software distributions of the current ROCm release fully supports this hardware.
```
"enables" sounds weird to me. "supports" sounds more natural to me, but not a blocker.